lib/new_relic/agent/llm.rb

```diff
@@ -3,9 +3,47 @@
 # frozen_string_literal: true
 
 require_relative 'llm/llm_event'
-require_relative 'llm/chat_completion'
 require_relative 'llm/chat_completion_message'
 require_relative 'llm/chat_completion_summary'
 require_relative 'llm/embedding'
-require_relative 'llm/feedback'
 require_relative 'llm/response_headers'
+
+module NewRelic
+  module Agent
+    class LLM
+      INPUT = 'input'
+      CONTENT = 'content'
+      SEGMENT_PATTERN = %r{Llm/.*/OpenAI/.*}.freeze
+
+      def self.instrumentation_enabled?
+        NewRelic::Agent.config[:'ai_monitoring.enabled']
+      end
+
+      # LLM content-related attributes are exempt from the 4095 byte limit
+      def self.exempt_event_attribute?(type, key)
+        return false unless instrumentation_enabled?
+
+        (type == NewRelic::Agent::Llm::Embedding::EVENT_NAME && key == INPUT) ||
+          (type == NewRelic::Agent::Llm::ChatCompletionMessage::EVENT_NAME && key == CONTENT)
+      end
+
+      def self.openai
```

Comment:
I still think the method should have a question mark:

```suggestion
      def self.openai?
```

(note that the memoization variable does not get the question mark)

Reply:
This is helpful. Should have a `?` as a true/false method and helps distinguish from the variable. Nice nice